site/src/pages/docs/server-annotated-service.mdx

```diff
@@ -1428,3 +1428,71 @@ for more information about RPC and HTTP service names, and how to use it.
 </Tip>
 
 [Publisher]: https://www.reactive-streams.org/reactive-streams-1.0.2-javadoc/org/reactivestreams/Publisher.html
+
+## Kotlin Coroutines Support
```

Comment:
We do not capitalize every word in headings. `Kotlin coroutines support` or `Kotlin Coroutines support`?

Reply:
I'll change to `Kotlin coroutines support`